README.md

```diff
@@ -1,60 +1,123 @@
-[![Coverage Status][3]][4]
-[![CodePolice][5]][6]
+<!-- Badges -->
+[![Actions Status][github_actions_badge]][github_actions_workflow_runs]
+[![CodePolice][codeclimate_maintainability_badge]][codeclimate_status_overview]
+[![Coverage Status][coveralls_badge]][coveralls_build]
 
-Last Revised: January 28, 2019
+Last Revised: 2023-01-08
 
-This README provides a quick overview of the tools used by Mushroom Observer
-(MO) and an overview of the other READMEs that accompany this file.  See
-<https://mushroomobserver.org/info/intro> for an introduction to the website
-itself.
+This README is an overview of
+- the tools used by Mushroom Observer (MO) and
+- the other READMEs in this repository.
+
+
+See
+<https://mushroomobserver.org/info/intro>
+for an introduction to the website itself.
 
-Tools
 -----
+## TOOLS
+### [git][git]
+MO uses [git][git] for version control and management of MO code.
+The source code is hosted on github. The URL for the MO source repository is
+https://github.com/MushroomObserver/mushroom-observer.
+### [Github Actions][Github Actions]
+We use [Github Actions][Github Actions]
+to implement Continuous Integration (CI).
+Pushing a commit to GitHub triggers a workflow run that includes:
+- testing with [minitest][minitest],
+- submitting the test results to [Coveralls][Coveralls]
+to obtain a test coverage report,
+- running [Codeclimate][CodeClimate] to get a quality control report.
+
+### [minitest][minitest]
+Our test suite uses [minitest][minitest] and extensions.
+See our [Gemfile][Gemfile] for more details.
+The test suite is run as part of our CI, and also can be run locally.
+
+### [Coveralls][Coveralls]
+We use [Coveralls][Coveralls] to obtain a
+[report about test coverage][coveralls_build].
+Coveralls is run as part of our CI, and also can be run locally.
+Its reports include total coverage and coverage for each line of code.
+The reports highlight newly uncovered lines
+so that we can maintain a high degree of coverage.
+
+### [Codeclimate][CodeClimate]
+We use [Codeclimate][CodeClimate]
+and plugins, like [RuboCop][RuboCop] and [Brakeman][Brakeman],
+to help maintain and improve code quality, including a consistent style.
+See our [Codeclimate configuration file](/.codeclimate.yml) for more details.
 
-Source code control: The source code for MO is currently managed using git
-and is hosted on github. The URL for the MO source repository is
-<https://github.com/MushroomObserver/mushroom-observer>.
+### [RuboCop][RuboCop]
+We use [RuboCop][RuboCop] and extensions to help with
+code quality and consistentcy of style.
+See our [RuboCop configuration file](/.rubocop.yml) for more details.
+[RuboCop][RuboCop] is run as part of our CI, and also can be run locally.
+(We are a long way from where we want to be.
+We have hundreds of offenses to fix.
+See our [Ruboco Todo file](.rubocop_todo.yml).)
 
+### [Brakeman][Brakeman]
+We use [Brakeman][Brakeman] to highight some security issues.
```

Comment:
I'm thinking about removing the explicit reference to Brakeman. It might be an invitation to hackers.